About/About.html

```diff
@@ -0,0 +1,135 @@
+            </div>
+            <div class="menu_item">
+                <a href="#">Login</a>
+            </div>
+        </div>
+    </div>
+    <div class="titlebar">
+        <div class="elements">
+            <img src="/images/Rectangle.png" alt="">
+            <img src="/images/twitter.png" alt="">
+            <p>English</p>
+            <p>My Page</p>
+            <p>Logout</p>
+        </div>
+    </div>
+    <!-- Nav bar -->
+    <nav id="nav_bar">
+        <button class="mob_menu" id="mobmenu"><img src="/images/mobile_menu 1.png" alt=""></button>
+        <div id="logo">
+            <a href="/Home/Home.html" class="cool-button">
+                <h1 class="cool-heading">GamerZportal</h1>
+            </a>
+            <div class="nav_elements">
+                <a href="/About/About.html">About</a>
+                <a href="#">News</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+            </div>
+        </div>
+    </nav>
+    <!-- main body -->
+    <section class="mainbody">
+        <div class="firstbod">
+            <div class="title">
+                <p id="welcome">
+                    "Welcome to GamerZportal"
+                </p>
+                <p id="desc_2">
+                    Game Forum 2023
+                </p>
+            </div>
+            <div class="description">
+                <p>
+                    Welcome to the Gaming Forum, the ultimate hub for gamers, game developers, and enthusiasts from
+                    around
+                    the
+                    world! Our virtual platform serves as a gathering place where experts, academics, and passionate
+                    gamers
+                    come
+                    together to discuss, explore, and celebrate the fascinating world of gaming.
+                </p>
+            </div>
+            <div class="contact">
+                <div class="part1">
+                    Please contact us per Email for any further questions about Gaming Forum 2023!
+                </div>
+                <div class="email">rdzaman20000@gmail.com</div>
+            </div>
+        </div>
+
+        <section class="part2">
+            <p class="desc5">Gamers Forum
+                Global Summit 2015 Logo</p>
+            <hr>
+            <p class="desc6" id="exc">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to
+                7, July.</p>
+        </section>
+        <section class="logo">
+            <p id="desc_7">
+                GamerZportal
+            </p>
+        </section>
+        <hr>
+        <section class="past_events">
+            <p class="desc7">See the Past Events</p>
+            <hr>
+            <p class="desc6">Take a look at the past two Gamers Summits which took place in Buenos Aires and in Warsaw
+            </p>
+            <div class="pe">
+                <div class="past_events1">
```

Comment:
- [ ] Please kindly center these images on mobile view and make use of the appropriate background image as shown in the provided project design below 👍 
- project design
![cim](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/63a79e6a-aa48-4ef5-bfb7-b30c4727f686)
- your design
![cims](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/3b673cc6-8c64-4d99-9c33-b96755d076cb)